### PR TITLE
Fixes for the newly merge feature/uberenv

### PR DESCRIPTION
**Pull request by adrienbernede**
2 fixes:
- In a moment of weakness I added "cuda" variant to umpire spack package... again. Umpire package inherits from CudaPackage.
- Restore symlink to blueos hostconfig files, deleted by mistake.

### CODE REVIEW

**Comment by adrienbernede** [comment]
LGTM

File: scripts/uberenv/packages/umpire/package.py [hunk 1]
```diff
@@ -80,7 +80,6 @@ class Umpire(CMakePackage, CudaPackage):
     variant('fortran', default=False, description='Build C/Fortran API')
     variant('c', default=True, description='Build C API')
     variant('numa', default=False, description='Enable NUMA support')
-    variant('cuda', default=False, description='Enable CUDA support')
```
Comment (davidbeckingsale) [inline scripts/uberenv/packages/umpire/package.py:83]: I assume this is because the `cuda` variant is inherited from `CudaPackage`?

Comment (adrienbernede) [reply scripts/uberenv/packages/umpire/package.py:83]: Yes

Comment (davidbeckingsale) [reply scripts/uberenv/packages/umpire/package.py:83]: Whoops, I didn't see your description :)

Comment (adrienbernede) [reply scripts/uberenv/packages/umpire/package.py:83]: We already discussed it, I made the mistake again :)

**Comment by adrienbernede** [comment]
LGTM

**Pull request merged**